hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -365,6 +357,7 @@ public void addDependencies(ServiceTarget target, ServiceBuilder<MonitorService>
 
         // bind the API to JNDI so other apps can use it, and prepare to build the binder service
         // Note that if we are running in host controller or similiar, JNDI binding is not available.
+        /*
```

Comment:
This JNDI stuff is going to be needed when third party devs want to write their own integration with Hawkular Inventory and Metrics. So we can't just rip it out. We would need some kind of code here to say , "if EAP6, don't bind to jndi, if EAP7 bind to jndi". We might want to consider aborting agent startup if the jndi-name attribute in the agent subsystem configuration (in standalone.xml) is set but we are on EAP6